### ISSUE TITLE
Exceptions when compiling for web

### ISSUE THREAD

**Issue by enricobenedos** [danvick/flutter_form_builder#211]
I'm trying to use the package for a web project. When I add the import `'package:flutter_form_builder/flutter_form_builder.dart'; `
line on my dart page file the project does not compile anymore.

```
enricobenedos@MBP-di-Enrico wamii_admin_app % flutter run -d chrome
Launching lib/main.dart on Chrome in debug mode...
Error creating flutter_chips_input|lib/flutter_chips_input.ddc.dill     
Error creating kernel summary for module:flutter_chips_input|lib/flutter_chips_input.ddc.dill

Response:--dart-sdk-summary=file:///Users/enricobenedos/SDKs/flutter/bin/cache/flutter_web_sdk/ke
rnel/flutter_ddc_sdk.dill
--output=/private/var/folders/td/0vt1vy1s24s4_ftgthkldlpw0000gn/T/scratch_spaceimzIS8/packages/fl
utter_chips_input/flutter_chips_input.ddc.dill
--packages-file=file:///var/folders/td/0vt1vy1s24s4_ftgthkldlpw0000gn/T/kernel_builder_Sh2K8Q/.pa
ckages --multi-root-scheme=org-dartlang-app --exclude-non-sources --summary-only --target=ddc
--libraries-file=file:///Users/enricobenedos/SDKs/flutter/bin/cache/flutter_web_sdk/libraries.jso
n --reuse-compiler-result --use-incremental-compiler
--used-inputs=/var/folders/td/0vt1vy1s24s4_ftgthkldlpw0000gn/T/kernel_builder_XJ9FKa/used_inputs.
txt --input-summary=org-dartlang-app:///packages/flutter/material.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/gestures.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/foundation.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/cupertino.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/widgets.ddc.dill
--input-summary=org-dartlang-app:///packages/vector_math/vector_math_64.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/semantics.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/rendering.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/animation.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/painting.ddc.dill
--input-summary=org-dartlang-app:///packages/collection/collection.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/src/gestures/arena.ddc.dill
--input-summary=org-dartlang-app:///packages/meta/meta.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/src/foundation/_bitfield_web.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/src/cupertino/action_sheet.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/src/widgets/actions.ddc.dill
--input-summary=org-dartlang-app:///packages/vector_math/hash.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/src/semantics/binding.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/src/rendering/animated_size.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/src/animation/animation.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/src/painting/_network_image_web.ddc.dill
--input-summary=org-dartlang-app:///packages/collection/src/algorithms.ddc.dill
--input-summary=org-dartlang-app:///packages/collection/src/utils.ddc.dill
--input-summary=org-dartlang-app:///packages/collection/src/iterable_zip.ddc.dill
--input-summary=org-dartlang-app:///packages/collection/src/comparators.ddc.dill
--input-summary=org-dartlang-app:///packages/collection/src/priority_queue.ddc.dill
--input-summary=org-dartlang-app:///packages/collection/src/canonicalized_map.ddc.dill
--input-summary=org-dartlang-app:///packages/typed_data/typed_buffers.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/physics.ddc.dill
--input-summary=org-dartlang-app:///packages/flutter/src/physics/clamped_simulation.ddc.dill
--source=package:flutter_chips_input/flutter_chips_input.dart
--source=package:flutter_chips_input/src/chips_input.dart
org-dartlang-app:///packages/flutter_chips_input/src/chips_input.dart:62:7: Error: The
non-abstract class 'ChipsInputState' is missing implementations for these members:
 - TextInputClient.currentTextEditingValue
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class ChipsInputState<T> extends State<ChipsInput<T>>
      ^^^^^^^^^^^^^^^
org-dartlang-app:///packages/flutter/src/services/text_input.dart: Context:
'TextInputClient.currentTextEditingValue' is defined here.



                                                                        
AssetNotFoundException: flutter_chips_input|lib/flutter_chips_input.ddc.dill
                                                                        
BuildStepCompletedException: Attempt to use a BuildStep after is has completed
Building application for the web...  
```
Flutter Doctor:
```
enricobenedos@MBP-di-Enrico wamii_admin_app % flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, v1.13.6-pre.16, on Mac OS X 10.15.2 19C57, locale en-IT)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 11.3)
[✓] Chrome - develop for the web
[✓] Android Studio (version 3.5)
[✓] VS Code (version 1.41.1)
[✓] Connected device (2 available)

• No issues found!
```
In my `pubspec.yaml` I've added only this package.

**Comment by danvick** [comment]
This should be fixed in v3.7.3

**Comment by enricobenedos** [comment]
Sorry for the late reply, I've directly tried with `flutter_form_builder: ^3.8.0+1` and it works!
